### PR TITLE
Tweak Forward and Section 3 README Anchors

### PR DESCRIPTION
**Pull request by kingthorin**
- Make the Foreward README be labeled zero.
- Lowercase the anchors in the section 3 README.
- Add 'Graphic Designers' to the credits.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: document/3-The_OWASP_Testing_Framework/README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # The OWASP Testing Framework
 
-3.1 [Overview](0-The_OWASP_Testing_Framework.md#Overview)
+3.1 [Overview](0-The_OWASP_Testing_Framework.md#overview)
 
-3.2 [Phase 1: Before Development Begins](0-The_OWASP_Testing_Framework.md#Phase-1-Before-Development-Begins)
+3.2 [Phase 1: Before Development Begins](0-The_OWASP_Testing_Framework.md#phase-1-before-development-begins)
```
Comment (rejahrehim) [inline document/3-The_OWASP_Testing_Framework/README.md:5]: The heading is different `## Phase 1: Before Development Begins` will this fragment links to this header? I think we need `#Phase-1:-Before-Development-Begins`

For all links with fragments

Comment (kingthorin) [reply document/3-The_OWASP_Testing_Framework/README.md:5]: Works fine on github:
https://github.com/OWASP/wstg/blob/af95d64a44c43762aaadd5543fe5d4e2932e2b18/document/3-The_OWASP_Testing_Framework/0-The_OWASP_Testing_Framework.md#phase-1-before-development-begins

And www:
https://owasp.org/www-project-web-security-testing-guide/latest/3-The_OWASP_Testing_Framework/0-The_OWASP_Testing_Framework.html#phase-1-before-development-begins

File: document/1-Frontispiece/README.md [hunk 2]
```diff
@@ -44,6 +44,10 @@ This document is released under the [Creative Commons 4.0 License](https://creat
 - Tal Argoni
 - Victoria Drake
 
+## Graphic Designers
+
+-Muhammed Anees
```
Comment (rejahrehim) [inline document/1-Frontispiece/README.md:49]: One more 
- Jishnu Vijayan C K

Comment (kingthorin) [reply document/1-Frontispiece/README.md:49]: Thanks, I had [asked ](https://github.com/OWASP/wstg/pull/409#issuecomment-611697513) but you're faster 👍

File: document/3-The_OWASP_Testing_Framework/README.md (hunk 1 above)
Comment (rejahrehim) [reply document/3-The_OWASP_Testing_Framework/README.md:5]: Checked in PDF build, Expected to work. But not :(

Comment (kingthorin) [reply document/3-The_OWASP_Testing_Framework/README.md:5]: Okay we'll have to come up with a new plan. @ThunderSon split it into individual files?

Comment (rejahrehim) [reply document/3-The_OWASP_Testing_Framework/README.md:5]: Will it work if we add `:` in Github and www like `#Phase-1:-Before-Development-Begins`

Comment (ThunderSon) [reply document/3-The_OWASP_Testing_Framework/README.md:5]: We can simply replace the `:` with ` - ` and then re-inject that in the hashstring. What do you think? I'd say splitting it into files won't make any difference as it's the `Phase X:` thing that is breaking it, so the subtitle header is the issue.

Comment (kingthorin) [reply document/3-The_OWASP_Testing_Framework/README.md:5]: Do we even need the colons? Is `Phase 2: During Definition and Design` really much clearer than `Phase 2 During Definition and Design`?

Comment (kingthorin) [reply document/3-The_OWASP_Testing_Framework/README.md:5]: Or we could do it like this: https://github.com/kingthorin/wstgjob#phase-2---break-stuff

Hopefully that'll work the same way on www.

**Comment by ThunderSon** [comment]
@rejahrehim please check if the PDF now builds properly based on my latest commit

**Comment by rejahrehim** [comment]
> based

**Comment by rejahrehim** [comment]
> @rejahrehim please check if the PDF now builds properly based on my latest commit

Working on it, Will send the latest build.

**Pull request merged**